# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/FramebufferAttachment.cpp
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/FramebufferAttachment.cpp
--- a/gfx/angle/checkout/src/libANGLE/FramebufferAttachment.cpp	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/FramebufferAttachment.cpp	Fri Nov 08 17:06:00 2019 +0000
@@ -178,12 +178,6 @@
     return mResource->getId();
 }
 
-const ImageIndex &FramebufferAttachment::getTextureImageIndex() const
-{
-    ASSERT(type() == GL_TEXTURE);
-    return mTarget.textureIndex();
-}
-
 TextureTarget FramebufferAttachment::cubeMapFace() const
 {
     ASSERT(mType == GL_TEXTURE);